_schemaV2.graphql

```diff
@@ -19006,6 +19006,7 @@ input updateCollectionInput {
   # The internal ID of the collection
   id: String!
   name: String!
+  shareableWithPartners: Boolean!
```

Comment:
I think this param can be optional. Otherwise the current UI that does not pass it (like Eigen) will fail.

Reply:
Good call 👍